Client/Clique/Common/Functions/Functions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Functions.swift
+//  Clique
+//
+//  Created by Infinum on 09.12.2022..
+//
+
+import UIKit
+
+enum Functions {
+    
+    enum Alerts {
+        static func alert(fwdMessage: String,viewController: UIViewController){
+            let alertController = UIAlertController(title: "", message: fwdMessage , preferredStyle: .alert)
+            let defaultAction = UIAlertAction(title: "OK", style: .cancel, handler: nil)
+            alertController.addAction(defaultAction)
+            viewController.present(alertController, animated: true, completion: nil)
+        }
+    }
+}
```

Comment:
This design is not good. I think I told you to make `UIViewController+Extension.swift` file or to put it in a extension of the same class, either way, remove Functions group, because you will not need it, and put it in a new file under Extensions group. This way method does not have to be static because the only place you should call alerts are from ViewController.

Also big point fix your namings and formatting. For example if the method name is alert that sends some kind of message, why are you using the name `fwdMessage` it makes no sense here. If the parameter name on the init of UIAlertController is message, why not be the same, also, why not utilize the title, it can be empty by default, like this:

`
extension UIViewController {

    func sendOkAlert(with title: String = "", message: String) {
        let alertController = UIAlertController(title: title, message: message , preferredStyle: .alert)
        let okAction = UIAlertAction(title: Constants.Alerts.ok, style: .cancel)
        alertController.addAction(okAction)
        present(alertController, animated: true)
    }

    func sendAlert(
        with title: String = "",
        message: String,
        actions: [UIAlertAction]
    ) {
        let alertController = UIAlertController(title: title, message: message , preferredStyle: .alert)
        let okAction = UIAlertAction(title: Constants.Alerts.ok, style: .cancel)
        actions.forEach { alertController.addAction($0) }
        present(alertController, animated: true)
    }
}
`